### DISCUSSION TITLE
Unable to Locate Workflows to Respond to Member Requests for GitHub Add-Ons

### DISCUSSION THREAD

**Discussion by w4jew-aprsfoundation** [community/community#138525] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My first post ever!

I logged into my GitHub account this morning and noticed that one individual with a GitHub for Teams account within my organization submitted a request for Copilot Business.

1. I have not found anywhere in the dashboard showing who initiated the request. Thankfully, it's a small account, and I know who it is—but if this was a larger organization, I would have no way of telling who initiated the request.

2. I also need help finding an option to reject the request.

3. Lastly, is there a way to create a policy to prevent members from submitting requests for Add-Ons? I can only imagine seeing many of these requests. I want to get on top of this while we're small.

I'm a n00b - so it's possible I just haven't come across it yet.

Any pointers would be greatly appreciated!!!

![image](https://github.com/user-attachments/assets/ec3fd12e-66ce-439d-a106-8a0779005ba1)

**Comment by tofr** [comment]
Loud SECOND.

**Comment by w4jew-aprsfoundation** [comment]
I figured this out on my own.

**Reply by tschmidtb51** [reply]
@w4jew-aprsfoundation Can you provide some hints how you solved 1-3?

**Reply by w4jew-aprsfoundation** [reply]
I'm trying to refresh my memory.... 

First - I had to change contexts from my user account to the organization. 

Your account needs to be assigned the "All Repository Admin" role and/or be an owner of the organization. 

https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization

In Organization settings, the Member Privileges section lets you enable or disable a user's ability to add apps (App Access Requests) and GitHub Apps. So, if you don't want to allow someone to even be able to request access to apps, you can modify these options to prevent them from submitting a request in the first place. 

However, this does _NOT_ disable the "Ask for a GitHub Copilot seat" option they're presented with when they accept an invite. 

<img width="1161" height="723" alt="image" src="https://github.com/user-attachments/assets/b217a02b-0458-4cbd-8d46-06050048dab2" />

You can disable Copilot entirely for the entire organization, but this will cancel the Copilot plan entirely and remove any existing seats and may be a little aggressive depending on what you plan to do.

There are a couple of other options:

1. Manage Copilot access for members in your organization settings. Settings -> Code, planning, and automation -> Copilot -> Access

It looks like you need to activate Copilot before you can manage user access - this is just dumb.

2. Review and approve/deny Copilot seat requests... Go to your Organization settings, then Access, then Requests from members.

You cannot deny a user request unless Copilot is enabled at the organization level. Which is also dumb.

When I first figured this out, I had Copilot enabled because GitHub had enabled trial/evaluation licensing on my account for 30-60 days. Since it was enabled, I could reject requests. But I don't have it enabled right now. I invited a user and had them select the "Request Copilot" option. 

I can see the user's request, but there's no option to cancel it.

<img width="1272" height="811" alt="image" src="https://github.com/user-attachments/assets/20ee46a3-2cc7-4a56-b85c-f7c5976ddda5" />


I would say this is worth opening a support case and/or an enhancement request to bring this to their attention. I would - but I don't really have cycles to stay on top of this. It seems like this is either intentional (trying to apply pressure to customers to sign up for Copilot), or a QA team that didn't realize the workflow is broken.

Sorry - that's probably not what you wanted to hear - but that's what I was able to glean from going through my records.

I hope it was at least somewhat helpful!
